src/pages/[SbtAddress].tsx

```diff
@@ -0,0 +1,282 @@
+            src={
+              sbtData
+                ? sbtData?.image.replace("ipfs://", "https://ipfs.io/ipfs/")
+                : "/visuals/sbtIllustration.webp"
+            }
+            alt="SBT image"
+          />
+          <div className={styles.textSection}>
+            {privateKey ? (
+              finalStep ? (
+                <>
+                  <h1 className={styles.title}>
+                    Your SBT was minted successfully
+                  </h1>
+                  <p className={styles.text}>
+                    <a
+                      href={
+                        process.env.NEXT_PUBLIC_NETWORK === "testnet"
+                          ? `https://testnet.starkscan.co/tx/${transactionHash}`
+                          : `https://starkscan.co/tx/${transactionHash}`
+                      }
+                      target="_blank"
+                      rel="noreferrer"
+                    >
+                      View transaction on Starkscan
+                    </a>
+                  </p>
+                </>
+              ) : (
+                <>
+                  <h1 className={styles.title}>It&apos;s almost done</h1>
+                  <div className={styles.identitySection}>
+                    <SelectIdentity
+                      tokenId={tokenId}
+                      changeTokenId={changeTokenId}
+                    />
+                    <Button onClick={updateCallData}>Mint my token</Button>
+                  </div>
+                </>
+              )
+            ) : (
+              <>
+                <h1 className={styles.title}>
+                  {sbtData && !loading && !error
+                    ? sbtData?.name
+                    : loading && !error
+                    ? "Fetching SBT data..."
+                    : "SBT not found"}
+                </h1>
+                <p className={styles.text}>
+                  {sbtData
+                    ? sbtData?.description
+                    : !loading && error
+                    ? `We could not find the SBT at ${contractAddress} address, please try another one.`
+                    : null}
+                </p>
+                <div className={styles.password}>
+                  {account && sbtData ? (
+                    <>
+                      <TextField
+                        fullWidth
+                        type="password"
+                        placeholder="Password"
+                        variant="outlined"
+                        onChange={(e) => changePassword(e.target.value)}
+                        required
+                      />
+                      <div className={styles.passwordBtn}>
+                        <Button onClick={() => handlePassword()}>Mint</Button>
+                      </div>
+                    </>
+                  ) : null}
+                </div>
+              </>
+            )}
+          </div>
+        </div>
+        <Connect />
+        {element}
```

Comment:
I guess this is for the loading part, It's better to set a boolean and use `isLoading ? <Element /> : null` don't you think ? 

If there is a specific reason I would love to have an explanation :)

Reply:
It's done that way on the page to deploy a SBT contract, that's why I did the same, but I can change it

Reply:
Oh ok, @Marchand-Nicolas stuff haha. 

Can you change it on both ?